sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,3 @@
+select "customers"
+from "customers"
+where "isActive" = 'false'
```

Comment:
The single quotes make this a string, which postgres then needs to convert to a boolean. Hopefully, it does it the way you intended. To remove the uncertainty, always use the proper data types. Boolean true and false should be specified without quotes.